cn.feishu.Feishu.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>cn.feishu.Feishu</id>
+  <update_contact>contact@feishu.cn</update_contact>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Feishu</name>
+  <summary>飞书(Feishu)</summary>
+  <description>
+    <p>
+    飞书(Feishu)
```

Comment:
Description should be descriptive not one word.

Reply:
I have changed the description.